dashboards/Experimental/MongoDB_Cluster_Summary_(new).json

```diff
@@ -3400,8 +3040,8 @@
       "30d"
     ]
   },
-  "timezone": "",
-  "title": "MongoDB Cluster Summary (New)",
+  "timezone": "browser",
```

Comment:
Should the timezone be set?

Reply:
Yes. as per the dashboard guidelines "local browser timezone for dashboards"